example/src/Pages/Layouts.js

```diff
@@ -8,34 +15,46 @@ import {
 } from "../../../lib/components/layouts";
 import {
   Button,
-  PageLoader,
   Pagination,
   Checkbox,
   Dropdown,
   Typography,
+  Table,
 } from "../../../lib/components";
-import {
-  MenuHorizontal,
-  Settings,
-  Plus,
-  Search,
-} from "@bigbinary/neeto-icons";
+import { TABLE_DATA, TABLE_HEADERS } from "../constants";
 
-const SidebarHandleIcon = ({size, color}) => {
-  return (
-    <svg
-      xmlns="http://www.w3.org/2000/svg"
-      width={size}
-      height={size}
-      fill="none"
-      viewBox="0 0 24 24"
-    >
-      <path stroke={color} strokeWidth="1.5" d="M3 7.25L21 7.25"></path>
-      <path stroke={color} strokeWidth="1.5" d="M3 11.25L15 11.25"></path>
-      <path stroke={color} strokeWidth="1.5" d="M3 15.25L11 15.25"></path>
-    </svg>
-  );
-}
+const useSortTable = (items, config = null) => {
```

Comment:
Can we export useSortTable from the table component? We do not need to repeat this code in the all the applications.